.ruby-version

```diff
@@ -1 +1 @@
-2.5.7
+2.6.0
```

Comment:
if you want to bump the default development version up, should we jump to the latest 2.6 rather than 2.6.0?

Reply:
My mistake - I had this confused with the VERSION file